doc/crud.md

```diff
@@ -53,6 +72,9 @@ extension data such as indexes (see [index-query.md](doc/index-query.md)):
 BOOL success = [manager deleteDatastoreNamed:@"my_datastore"
                                        error:&error];
 ```
+```swift
+try manager.deleteDatastoreNamed("my_datastore")
```

Comment:
There is no `boolean` value coming out from it in swift, because of the error param it gets converted to

` public func deleteDatastoreNamed(name) -> Void throws`